tests/helper/index.ts

```diff
@@ -0,0 +1,27 @@
+import { FastifyInstance } from "fastify";
+import { initializeDatabase } from "../../src/database";
+import { initializeServer } from "../../src/server";
+
+/**
+ * Helper function to set up all the needed environment for integration test.
+ * With this function, it will set DB and server at once.
+ */
+export function build(): FastifyInstance {
+  // Set up server. This instance will be the return
+  // for jest to access and manipulate its APIs
+  const app = initializeServer();
+
+  // Set up DB with mocked environment valuables.
+  beforeAll(async () => {
+    process.env.POSTGRES_USER = "root";
+    process.env.POSTGRES_PASSWORD = "root";
+    process.env.POSTGRES_DB = "trello-clone";
+    await initializeDatabase();
+  });
+
+  afterAll(async () => {
+    await app.close();
+  });
+
+  return app;
```

Comment:
This helper function is basically equivalent to `index.ts`. Instead of running `initializeApp()`, jest would set up and run DB and server from this function and jest can access Fastify's instance to mock-up requests with `.inject`